bin/ard_pbs

```diff
@@ -99,7 +99,7 @@ def _submit_multiple(scattered, env, batch_logdir, batch_outdir, pkgdir,
             raise
 
         if hasattr(raw_output, 'decode'):
-            matches = re.match('^(?P<nci_job_id>\d{7,}.r-man2)$', raw_output.decode('utf-8'))
+            matches = re.match('^(?P<nci_job_id>\d+.gadi-pbs)$', raw_output.decode('utf-8'))
```

Comment:
Looks good.

Note that the regexp that was here is slightly misleading: the dot is matching any character, not just a dot. And the backslashed-d only works because it's not a python escape character in current Python versions. It's slightly better practice to use raw strings and a literal dot, eg:
```suggestion
            matches = re.match(r'^(?P<nci_job_id>\d+\.gadi-pbs)$', raw_output.decode('utf-8'))
```
(lint tools will complain about these)

But neither of these matter in practice in this case, so I'm approving anyway.

Reply:
Simple to fix, I'll update.